features/step_definitions/add_review_steps.rb

```diff
@@ -0,0 +1,27 @@
+Given(/^I am logged in$/) do
+  click_link('Sign Up')
+  fill_in('user_email', with: 'user1@tamu.edu')
+  fill_in('user_password', with: '12345')
+  click_button('Save')
```

Comment:
Due to the changes, Dawson made this will no longer work.  I would still recommend merging so we can fully see what is working and not working with the tests with everyone's changes together.